tools/filamesh/CMakeLists.txt

```diff
@@ -24,15 +24,15 @@ target_link_libraries(${TARGET} PRIVATE getopt filameshio meshoptimizer)
 # ==================================================================================================
 # Compile options and optimizations
 # ==================================================================================================
-target_compile_options(${TARGET} PRIVATE
-        -Wno-deprecated-register
-)
+if(NOT MSVC)
+    target_compile_options(${TARGET} PRIVATE -Wno-deprecated-register)
```

Comment:
identation

Reply:
I fixed the indentation. I'm happy to fix things but feel free to make changes if it is easier.